robottelo/ui/discoveryrules.py

```diff
@@ -68,10 +67,6 @@ def search(self, name):
         """Searches existing discovery rule from UI. It is necessary to use
         custom search as we don't have both search bar and search button there.
         """
-        if not bz_bug_is_open(1233135):
```

Comment:
According to BZ, this issue would be tracked via redmine, that's where we can use `rm_bug_is_open` :D